app/controllers/application_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) do |u|
+      u.permit(:name, :email, :password, :password_confirmation)
+    end
+    devise_parameter_sanitizer.permit(:account_update) do |u|
+      u.permit(:name, :email, :password, :current_password, :password_confirmation)
+    end
+  end
```

Comment:
- [ ] [OPTIONAL] I wasn't able to sign up through sign up page please make sure to fix that in order your app will be working properly